OpenBSD doesnt have librt

make sure rpath is set in the resulting binary for libinotify.so

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -41,7 +41,7 @@ xkb = dependency('xkbcommon')
 fontconfig = dependency('fontconfig')
 freetype = dependency('freetype2')
 threads = dependency('threads')
-rt = cc.find_library('rt')
+# rt = cc.find_library('rt')
 
 # optional dependencies: file formats support
 exr = dependency('OpenEXR', version: '>=3.1', required: get_option('exr'))
@@ -234,7 +234,7 @@ executable(
   sources,
   dependencies: [
     # runtime
-    rt,
+    # rt,
     threads,
     wlcln,
     epoll,
@@ -256,5 +256,6 @@ executable(
     tiff,
     webp, webp_demux,
   ],
+  install_rpath : '${LOCALBASE}/lib/inotify',
   install: true
 )
